tests/baselines/reference/docker/xterm.js.log

```diff
@@ -1,9 +1,24 @@
-Exit Code: 0
+Exit Code: 2
 Standard output:
 
 > xterm@X.X.X build /xtermjs
 > tsc -b ./tsconfig.all.json
+src/browser/Clipboard.ts(85,5): error TS2322: Type 'null' is not assignable to type 'string'.
```

Comment:
These are due to a DOM update that removed `null` as a valid type for some members of `CSSStyleDeclaration`.